ui/src/turing/components/configuration/StandardEnsemblerConfigDetails.js

```diff
@@ -14,7 +14,7 @@ const StandardEnsemblerConfigDetailsComponent = ({ projectId, routes, routeNameP
     <ConfigProvider>
       <EuiFlexGroup direction="row" wrap>
         <EuiFlexItem grow={1} className="euiFlexItem--smallPanel">
-          <ConfigSectionPanel title="Route Selection" className="experimentSummaryPanel">
```

Comment:
Do you think we can simply apply the `white-space: normal !important;` to the existing `experimentSummaryPanel` style ? Do you see any downside?

Reply:
Let me hunt around and see where the existing style is used 👀

Reply:
Seems like it's only used here: 
![Screenshot 2022-09-27 at 5 05 11 PM](https://user-images.githubusercontent.com/36802364/192484054-044cc619-e888-4c37-a5bc-20cc7fc6fb49.png)

And since the number of experiments won't be anything as long as the route name path, I guess we can safely apply that to the `experimentSummaryPanel` as you suggested 🙂